Library: Add OCInferno (#257)
SpecterOps/bloodhound-docs: docs/opengraph/library.mdx

**Repo**
- https://github.com/SpecterOps/OktaHound

</Accordion>
<Accordion title="Oracle Cloud Infrastructure" icon="people-group">

#### <Icon icon="people-group" size={22}/> OCInferno

**Description**

> OCInferno (O-C-Inferno) is an OCI offensive security assessment framework for workspace-driven credential handling, service enumeration, artifact download, and graph-based attack-path analysis. It includes a module to generate a custom OpenGraph output which can be fed into BloodHound, as shown below, to map privilege-escalation paths.

Reviewer: ```suggestion
OCInferno (O-C-Inferno) is an OCI offensive security assessment framework for workspace-driven credential handling, service enumeration, artifact download, and graph-based attack-path analysis. It includes a module to generate a custom OpenGraph data payload that you can upload to BloodHound to map privilege-escalation paths.
```
